scripts/inspect_kernel_pytorch_profiler.py

```diff
@@ -97,7 +97,9 @@ def get_torch_profiler_info(ref_arch_src: str,
 
         profiler_output = prof.key_averages().table(sort_by='cuda_time_total', 
                                                     row_limit=table_row_limit)
-        
+    
+    # delete the tempfile
+    os.remove(tempfile_path)
```

Comment:
favor using `tempfile` package instead, `os.remove()` won't return an error code if this command fails